src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/UsePropertiesWhereAppropriateTests.cs

```diff
@@ -117,6 +117,11 @@ public IEnumerator GetEnumerator()
         return null;
     }
 
+    public ref string GetPinnableReference() // If the method isn't ref-returning, there will be a diagnostic.
```

Comment:
Could you add a test for `readonly ref` (or the other way around, I can't recall the expected order). Also, could you add a test for a managed type.

Reply:
I think you missed the managed type case.